cocos/base/CCEventDispatcher.cpp

```diff
@@ -973,7 +973,30 @@ void EventDispatcher::dispatchTouchEvent(EventTouch* event)
     //
     if (oneByOneListeners)
     {
-        auto mutableTouchesIter = mutableTouches.begin();
+        if (event->getEventCode() == EventTouch::EventCode::BEGAN)
+        {
+            auto fixedPriorityListeners = oneByOneListeners->getFixedPriorityListeners();
+            auto sceneGraphPriorityListeners = oneByOneListeners->getSceneGraphPriorityListeners();
+            if (fixedPriorityListeners != nullptr)
+            {
+                for (auto l : (*fixedPriorityListeners))
+                {
+                    EventListenerTouchOneByOne* listener = static_cast<EventListenerTouchOneByOne*>(l);
+                    listener->_claimedTouches.clear();
```

Comment:
@dadidzf , if clear `_claimedTouches`, it may break the `multi-touch` logic of using EventListenerTouchOneByOne. I will check `TouchTest` to see whether it works after this fix.